libraries/objectDefaultFiles/object.js

```diff
@@ -2151,6 +2158,36 @@
                 }
             })
         }
+        
+        this.toggleMeasureMode = function(boolean) {
+            postDataToParent({
+                toggleMeasureMode: boolean
+            });
+        }
+
+        this.toggleCrossRotation = function(boolean) {
+            postDataToParent({
+                toggleCrossRotation: boolean
+            });
+        }
+        
+        this.toggleCloseLoop = function(boolean) {
+            postDataToParent({
+                toggleCloseLoop: boolean
+            });
+        }
+        
+        this.turnMeasureMapUI = function(boolean) {
+            postDataToParent({
+                turnMeasureMapUI: boolean
+            });
+        }
+
+        this.toggleMeasureMapUI = function() {
+            postDataToParent({
+                toggleMeasureMapUI: true
+            });
+        }
```

Comment:
Can you add a comment to each of these APIs documenting what they do? It will be useful in future to not forget what these do.